Comparing lossless versus lossy compression for Windows users

JeremyPike opened the pull request:
The _Comparing lossless versus lossy compression_ exercise does not work for Windows users who do not have a bash terminal with zip installed (I tried Git for Windows but zip not included). Here I've added a potential command line solution using Windows PowerShell.

mkcor commented on episodes/02-image-basics.md:
```diff
@@ -881,20 +881,22 @@ two orders of magnitude smaller than the bitmap version.
 
 Having created the compressed file,
-use the `ls -al` command to display the contents of the directory.
+use the `ls -al` command (`dir` with PowerShell) to display the contents of the directory.
```
Can you use the `dir` command with options (equivalent to `-al`) in PowerShell?

JeremyPike replied:
@mkcor I don't think additional options are needed as `dir` outputs in a format similar to `ls -l` by default, importantly including file size (at least for me).

bobturneruk replied:
I think `dir` and `ls` are synonymous for `Get-ChildItem` in PowerShell. These return these columns:
```
Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
```
Length I believe is file size in bytes.

mkcor replied:
Ok, good to know. And how about displaying hidden files and folders (equivalent to `-a`)?

JeremyPike replied:
There's a -Force flag which I think does this but it isn't really needed here as the files we are interested in are not hidden

tobyhodges replied:
@mkcor and @bobturneruk I am inclined to merge this. Please speak up in the next few days if you would like to propose further changes.

mkcor replied:
Ok, then I would suggest removing the `-a` flag here.

JeremyPike commented:
Thanks @tobyhodges, I'll leave this for now and come back to it once Ive gone through the rest of the course to see if Powershell would be useful elsewhere. I think modern versions of Windows (10 and newer) come shipped with some version of Powershell so in most cases no install would be required (but a link just in case wouldn't hurt).

JeremyPike commented:
@tobyhodges so I only found one other terminal command in the course which I've added the Powershell equivalent command for. I've also added a note in the setup instructions but I would be surprised if anyone didn't have it installed by default. Note in [JupterLab Windows users get Powershell](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html) if they open a terminal so that works nicely too :)

mkcor commented on episodes/03-skimage-images.md:
```diff
@@ -189,8 +189,8 @@ Images may appear the same size in jupyter,
 but you can see the size difference by comparing the scales for each.
 You can also see the differnce in file storage size on disk by
```
```suggestion
You can also see the difference in file storage size on disk by
```

mkcor commented on episodes/03-skimage-images.md:
```diff
@@ -189,8 +189,8 @@ Images may appear the same size in jupyter,
 hovering your mouse cursor over the original
-and the new file in the jupyter file browser, using `ls -l` in your shell,
-or the OS file browser if it is configured to show file sizes.
+and the new file in the jupyter file browser, using `ls -l` in your shell 
```
```suggestion
and the new files in the Jupyter file browser, using `ls -l` in your shell 
```

mkcor commented on episodes/03-skimage-images.md:
```diff
@@ -189,8 +189,8 @@ Images may appear the same size in jupyter,
-and the new file in the jupyter file browser, using `ls -l` in your shell,
-or the OS file browser if it is configured to show file sizes.
+and the new file in the jupyter file browser, using `ls -l` in your shell 
+(`dir` with Windows PowerShell), or the OS file browser if it is configured to show file sizes.
```
```suggestion
(`dir` with Windows PowerShell), or viewing file sizes in the OS file browser if it is configured so.
```
Because it goes like "by hovering ..., using ..., or ..." (right?).

mkcor commented on episodes/02-image-basics.md:
```diff
@@ -881,20 +881,22 @@ two orders of magnitude smaller than the bitmap version.
 
 Having created the compressed file,
-use the `ls -al` command to display the contents of the directory.
+use the `ls -al` command (`dir` with PowerShell) to display the contents of the directory.
```
```suggestion
use the `ls -l` command (`dir` with PowerShell) to display the contents of the directory.
```

mkcor reviewed (approved):
Fine by me, @tobyhodges! I caught a few details which this PR only brought to my attention now.

JeremyPike commented:
> Fine by me, @tobyhodges! I caught a few details which this PR only brought to my attention now.

Thanks, all suggestions made :)
